docs/angular/guides/setup-mfe-with-angular.md

```diff
@@ -0,0 +1,393 @@
+# How to setup a Micro Frontend with Angular and Nx
+
+[Webpack 5]() introduced a [Module Federation Plugin]() enabling multiple, independently built and deployed bundles of code to form a single application. This is the foundation of Micro Frontend Architecture and the Module Federation Plugin makes implementing such an architecture much simpler.  
+With Angular 12 adding support for Webpack 5 it increases the viability of scaffolding a Micro Frontend architecture with Angular.
+
+There are some challenges that exist, such as being able to be able to pass a custom Webpack configuration file, which is required to add the Module Federation configuration, to Angular's compiler.
+
+We recently made changes to our `@nrwl/angular:webpack-browser` executor that allows you to supply a path to a custom Webpack configuration file which we then merge with Angular's default Webpack configuration before telling Angular to use this merged configuration to build the Angular application.
+
+We also made some changes to our `@nrwl/angular:app` generator to aid in the scaffolding of Module Federation configuration required for setting up a Micro Frontend Architecture.
+
+Therefore, using Nx it can be fairly straightforward to scaffold and build a Micro Frontend Architecture from a monorepo with all the additional benefits of Nx.
+
+In this article, I'll show you to how setup a Micro Frontend Architecture with Nx and Angular.
+
+## What we'll build
+
+We will put together a simple Admin Dashboard application that requires a user to log in to view the protected content.  
+To achieve this we will have two apps:
+
+- Admin Dashboard app
+  - This will contain the layout of the dashboard and the content that can only be viewed by an authenticated and authorized user.
+- Login app
+  - This will contain the forms and logic for authenticating and authorizing a user
+
+When the user tries to view a protected page within the Admin Dashboard, if they are not authenticated we will present them with a login form so that they can authenticate and view the contents of the page.
+
+The end result should look something like this:
+
+## First steps
+
+### Create an Nx Workspace
+
+To start with, we need to create a new Nx Workspace. We can do this easily with:  
+**npm**  
+`npx create-nx-workspace ng-mfe`
+
+**Yarn**  
+`yarn create nx-workspace ng-mfe --packageManager=yarn`
+
+You'll be prompted for a preset. We recommend selecting `empty` as it will allow you finer control over your workspace configuration.
+
+You'll also be prompted if you would like to setup Nx Cloud. For this tutorial select `No`, however, I highly recommend that you read more about it [here](https://nx.app/).
+
+### Add the Angular Plugin
```

Comment:
Same as above.

Reply:
My argument for this is the same as above also lol